package.json

```diff
@@ -139,15 +139,16 @@
     "@radix-ui/react-accessible-icon": "1.0.3",
     "@radix-ui/react-dropdown-menu": "2.0.6",
     "@radix-ui/react-select": "2.0.0",
-    "@radix-ui/themes": "2.0.2",
+    "@radix-ui/react-tabs": "1.0.4",
+    "@radix-ui/themes": "2.0.3",
     "@reduxjs/toolkit": "1.9.6",
     "@scure/base": "1.1.3",
     "@scure/bip32": "1.3.2",
     "@scure/bip39": "1.2.1",
     "@scure/btc-signer": "1.1.0",
-    "@segment/analytics-next": "1.56.0",
-    "@sentry/react": "7.86.0",
-    "@sentry/tracing": "7.72.0",
+    "@segment/analytics-next": "1.62.1",
```

Comment:
How do you determine what to update? Do you do something like `yarn upgrade`?

Reply:
Manually going through the packages I believe are safe to update

Reply:
Thank. Asking as it seems a tedious manual job. Fair play